.gitignore

```diff
@@ -5,3 +5,4 @@ vendor
 classes/log/error_log.txt
 composer.phar
 .idea/
+lib/*/docs/examples/*_test.json
```

Comment:
I've pushed another commit to address this issue.

* The change to `.gitignore` is no longer necessary so I've backed it out.
* The automatic generation of the file name of the example event JSON files by the PHPUnit tests has been corrected (changed from replacing `Test` to replacing `_test`).
* Example event JSON files themselves (in `lib/*/docs/examples/`) have all been renamed